core/src/include/jcr.h

```diff
@@ -99,13 +99,13 @@ class JobControlRecord {
   void IncUseCount(void)
   {
     lock();
-    _use_count++;
```

Comment:
what exactly lead to this change?
If it was the compiler warning, then you can just `++x` instead of `x++`.

Reply:
"compound assignements (op=) and post/pre increment/decrement operations ( -- , ++ ) have been deprecated in C++20 "
Since `_use_count` is a volatile, those increment/decrement operations are deprecated and produce a warning.
Doing `variable = variable + 1` is still allowed.

plenty of online content talk about it